app/controllers/marks_graders_controller.rb

```diff
@@ -113,7 +116,7 @@ def global_actions
   def students_with_assoc
     Student.includes(
       :section,
-      grade_entry_students: { grade_entry_student_tas: :ta })
+      :grade_entry_students => {grade_entry_student_tas: :ta})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.